doc/source/whatsnew/v0.19.0.txt

```diff
@@ -619,6 +619,36 @@ Furthermore:
 - Passing duplicated ``percentiles`` will now raise a ``ValueError``.
 - Bug in ``.describe()`` on a DataFrame with a mixed-dtype column index, which would previously raise a ``TypeError`` (:issue:`13288`)
 
+.. _whatsnew_0190.api.perioddtype:
+
+``PeriodIndex`` now has ``period`` dtype
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``PeriodIndex`` now has its own ``period`` dtype. ``period`` dtype is a
+pandas extension dtype which extends NumPy dtype like ``"category"``. (:issue:`13941`)
+
+Previous Behavior:
+
+.. code-block:: ipython
+
+   In [1]: pi = pd.PeriodIndex(['2016-08-01'], freq='D')
+   In [2]: pi
+   Out[2]: PeriodIndex(['2016-08-01'], dtype='int64', freq='D')
+   In [3]: pd.types.api.is_integer_dtype(pi)
```

Comment:
This is not the current public entry point (although depends on outcome of discussion in https://github.com/pydata/pandas/issues/13634), I think `pd.api.types.is_..` is